perception/conda/recipe/build.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+which python
```

Comment:
`set -e` is useful at the beginning, that way the script will stop as soon as a command fails.

Reply:
circleci uses `set -eo pipefail` to catch errors in a sequence of pipes